Content/0.4 - Settings/settings.json

```diff
@@ -9,6 +9,7 @@
 	"settings_backups": "Backups",
 	"settings_importFromLegacyWallet": "Import from a Legacy Wallet",
 	"settings_ledgerHardwareWallets": "Ledger Hardware Wallets",
+	"settings_depositGuarantees": "Deposit Guarantees",
```

Comment:
Should we also add the subtitle string - "Set your default guaranteed minimum for estimated deposits" -> [ref](https://app.zeplin.io/project/639c7e7ab391bb15ac5d25fa/screen/63f7a6d4024b9109d6833582)

Reply:
> Should we also add the subtitle string - "Set your default guaranteed minimum for estimated deposits" -> [ref](https://app.zeplin.io/project/639c7e7ab391bb15ac5d25fa/screen/63f7a6d4024b9109d6833582)

Good point I focus on iOS and subtitle wasn't there. Will do now 👍